brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post Grid - Link to complete box redirect to last post when image background set to Top.

### Description

- Trello - https://trello.com/c/54sH1RGk/294-fix-post-grid-link-to-complete-box-when-image-background-set-to-top

### Screenshots

- https://share.getcloudapp.com/E0uYLWwK

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Post grid.
- Set image bg to Background and Link to complete box enable. check each post redirect to a specific post.
- Set image bg to Top and check each post redirect to a specific post.

### Checklist:
